example/src/RefBar.js

```diff
@@ -0,0 +1,26 @@
+import { useEffect, useRef, useState } from "react"
+import ProgressBar from '@ramonak/react-progress-bar'
+
+const RefBar = () => {
+    const ref = useRef()
+    const [size, setSize] = useState()
+
+    useEffect(() => {
+        console.log(ref.current)
```

Comment:
ref.current - undefined

Reply:
I was originally going to use the progress bar to show the progress of a video. I was going to do the mouse x / element width to find out what part of the movie the user would seek to. 

I have switched to using the ant design progress bar because my project was using ant design. 

I also can't think of any other use cases for this, so you can close the pull request.